pkgs/development/python-modules/aplpy/default.nix

```diff
@@ -81,5 +81,7 @@ buildPythonPackage rec {
     homepage = "http://aplpy.github.io";
     license = licenses.mit;
     maintainers = with maintainers; [ smaret ];
+    # Tests fail
```

Comment:
We should at least mention what tests. If only a small fraction fails we usually just ignore that test to not make the package unavilable.